ClinicWeb/src/main/java/pl/infoshare/clinicweb/user/service/UserService.java

```diff
@@ -66,5 +127,15 @@ public boolean isUserAlreadyRegistered(String email) {
 
     }
 
+    public UserDto getLoggedInUser() {
+
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        String email = authentication.getName();
+
+        return userRepository.findUserByEmail(email)
+                .map(userMapper::toDto)
+                .orElseThrow(() -> new RuntimeException("User was not found"));
```

Comment:
1. Warto dodać wyszukiwany email do logowanej wiadomości "User was not found with given email: %s"
2. Warto utworzyć dedykowany exception np. UserNotFoundByEmialException. ,albo gotowe rozwiązanie EntitiNotFoundException.